eval.cc

```diff
@@ -426,6 +426,15 @@ void Evaluator::EvalInclude(const IncludeStmt* stmt) {
     ScopedTerminator st(pat);
     vector<string>* files;
     Glob(pat.data(), &files);
+    if (files->size() == 0) {
+      for (auto inc_path : g_flags.include_dirs) {
+        auto to_check = (inc_path + '/' + pat.data());
+        LOG("searching for %s in : %s", pat.data(), inc_path.c_str());
+        Glob(to_check.c_str(), &files);
```

Comment:
It seems that in GNU make, the include directories are not searched using globs. You can test this with:

root/
   Makefile - include foo*.mk
   dir1
      fooa.mk

make -Idir1 gives "foo*.mk: No such file or directory", but if you change the include to fooa.mk explicitly, it works.